sdk/confidentialledger/azure-security-confidentialledger/src/main/java/com/azure/security/confidentialledger/ConfidentialLedgerClient.java

```diff
@@ -0,0 +1,463 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.security.confidentialledger;
+
+import com.azure.core.annotation.Generated;
+import com.azure.core.annotation.ReturnType;
+import com.azure.core.annotation.ServiceClient;
+import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.exception.ClientAuthenticationException;
+import com.azure.core.exception.HttpResponseException;
+import com.azure.core.exception.ResourceModifiedException;
+import com.azure.core.exception.ResourceNotFoundException;
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.core.http.rest.RequestOptions;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.BinaryData;
+
+/** Initializes a new instance of the synchronous ConfidentialLedgerClient type. */
+@ServiceClient(builder = ConfidentialLedgerClientBuilder.class)
+public final class ConfidentialLedgerClient {
+    @Generated private final ConfidentialLedgerAsyncClient client;
+
+    /**
+     * Initializes an instance of ConfidentialLedgerClient class.
+     *
+     * @param client the async client.
+     */
+    @Generated
+    ConfidentialLedgerClient(ConfidentialLedgerAsyncClient client) {
+        this.client = client;
+    }
+
+    /**
+     * The constitution is a script that assesses and applies proposals from consortium members.
+     *
+     * <p><strong>Response Body Schema</strong>
+     *
+     * <pre>{@code
+     * {
+     *     digest: String
+     *     script: String
+     * }
+     * }</pre>
+     *
+     * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @return the governance script for the application along with {@link Response}.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<BinaryData> getConstitutionWithResponse(RequestOptions requestOptions) {
```

Comment:
These `withResponse` methods should have a way to pass in `Context`?

cc: @srnagar

Reply:
For DPG, the context can be set in `RequestOptions`. So, we don't need explicit arg in the client method to set the context.

`new RequestOptions().setContext(context)`.